src/gateways/translators/IApiResponse.ts

```diff
@@ -0,0 +1,17 @@
+export interface IApiResponse {
+  meta: Record<string, any>;
+  data: Record<string, any> | Record<string, any>[];
+  links: {
+    self: string;
+    next: string;
+    last: string;
+  };
+  included: {
+    type: string;
+    id: string;
+    attributes: Record<string, any>;
+    links: {
+      self: string;
+    };
+  }[];
+}
```

Comment:
Generic to _any_ API response (I believe).  So the `data` property can either be itself a key-value pair or an array of key-value pairs.  I think this pattern will be common across all endpoints (`/beacon/:id`, `/beacons`, but also `/owners`, `/owner/:id` etc.) @Ninamma @tm1303 ?